pkgs/by-name/ca/cargo-tauri/package.nix

```
@@ -12,63 +12,70 @@
-  };
+    src = fetchFromGitHub {
+      owner = "tauri-apps";
+      repo = "tauri";
+      rev = "2b61447dfc167ec11724f99671bf9e2de0bf6768";
+      hash = "sha256-gKG7olZuTCkW+SKI3FVZqgS6Pp5hFemRJshdma8rpyg=";
+    };
 
-  # Manually specify the sourceRoot since this crate depends on other crates in the workspace. Relevant info at
-  # https://discourse.nixos.org/t/difficulty-using-buildrustpackage-with-a-src-containing-multiple-cargo-workspaces/10202
-  sourceRoot = "${src.name}/tooling/cli";
+    # Manually specify the sourceRoot since this crate depends on other crates in the workspace. Relevant info at
+    # https://discourse.nixos.org/t/difficulty-using-buildrustpackage-with-a-src-containing-multiple-cargo-workspaces/10202
+    sourceRoot = "${src.name}/tooling/cli";
 
-  cargoHash = "sha256-VXg/dAhwPTSrLwJm8HNzAi/sVF9RqgpHIF3PZe1LjSA=";
+    cargoHash = "sha256-VXg/dAhwPTSrLwJm8HNzAi/sVF9RqgpHIF3PZe1LjSA=";
 
-  nativeBuildInputs = [ pkg-config ];
+    nativeBuildInputs = [ pkg-config ];
 
-  buildInputs =
-    [ openssl ]
-    ++ lib.optionals stdenv.hostPlatform.isLinux [
-      gtk3
-      libsoup
-      webkitgtk_4_0
-    ]
-    ++ lib.optionals stdenv.hostPlatform.isDarwin (
-      with darwin.apple_sdk.frameworks;
-      [
-        CoreServices
-        Security
-        SystemConfiguration
+    buildInputs =
+      [ openssl ]
+      ++ lib.optionals stdenv.hostPlatform.isLinux [
+        gtk3
+        libsoup
+        webkitgtk_4_0
       ]
-    );
-
-  passthru = {
-    # See ./doc/hooks/tauri.section.md
-    hook = callPackage ./hook.nix { };
+      ++ lib.optionals stdenv.hostPlatform.isDarwin (
+        with darwin.apple_sdk.frameworks;
+        [
+          CoreServices
+          Security
+          SystemConfiguration
+        ]
+      );
 
-    tests = {
-      setupHooks = callPackage ./test-app.nix { };
+    meta = {
+      description = "Build smaller, faster, and more secure desktop applications with a web frontend";
+      homepage = "https://tauri.app/";
+      changelog = "https://github.com/tauri-apps/tauri/releases/tag/tauri-v${version}";
+      license = with lib.licenses; [
+        asl20 # or
+        mit
+      ];
+      maintainers = with lib.maintainers; [
+        dit7ya
+        getchoo
+        happysalada
+      ];
+      mainProgram = "cargo-tauri";
     };
   };
+in
 
-  meta = {
-    description = "Build smaller, faster, and more secure desktop applications with a web frontend";
-    homepage = "https://tauri.app/";
-    changelog = "https://github.com/tauri-apps/tauri/releases/tag/tauri-v${version}";
-    license = with lib.licenses; [
-      asl20 # or
-      mit
-    ];
-    maintainers = with lib.maintainers; [
-      dit7ya
-      getchoo
-      happysalada
-    ];
-    mainProgram = "cargo-tauri";
-  };
-}
+# We use overrideAttrs, since we can't directly use the finalAttrs pattern with buildRustPackage
+cargo-tauri.overrideAttrs (
+  finalAttrs: prevAttrs: {
+    passthru = {
```

```suggestion
    passthru = lib.recursiveUpdate prevAttrs.passthru or { } {
```
We should make sure we don't override any previous values set in the future

---

I don't think there's any reason to have a passthru in two places for the same package.

The problem will instead be what to do with the passthru tests with `cargo-tauri_1`... 
Though handling that would most likely not be the job of the `_2` package. `_1` could probably just `removeAttrs` it or override it to the `_1` example package.